orttraining/orttraining/models/runner/training_runner.h

```diff
@@ -55,6 +57,9 @@ class TrainingRunner {
 
     MapStringToString input_name_map;
 
+    // generic properties for storing perf metrics
+    mutable MapStringToString perf_properties;
```

Comment:
why does it need to be mutable?

Reply:
the compile would failed when I call the Get...(), let me get the message details ...